okio/src/commonMain/kotlin/okio/ByteString.kt

```diff
@@ -144,7 +144,6 @@ internal constructor(data: ByteArray) : Comparable<ByteString> {
   override fun toString(): String
 
   companion object {
-    // TODO null in JS and Native - https://youtrack.jetbrains.com/issue/KT-26497
```

Comment:
I was able to work around this issue by removing `COMMON_EMPTY` and constructing the ByteString directly in the actual implementations.